File: QingLian-app-root/src/main/java/com/yychainsaw/pojo/dto/WorkoutRecordDTO.java
package com.yychainsaw.pojo.dto;

import jakarta.validation.constraints.NotNull;
import lombok.Data;


@Data
public class WorkoutRecordDTO {
    @NotNull(message = "运动记录不能为空")
    private Integer durationSeconds;
    @NotNull(message = "运动记录不能为空")
    private Integer caloriesBurned;

    private String notes;
}

File: QingLian-app-android/app/src/main/java/com/yychainsaw/qinglianapp/data/model/dto/WorkoutRecordDTO.kt
package com.yychainsaw.qinglianapp.data.model.dto

data class WorkoutRecordDTO(
    val durationSeconds: Int,
    val caloriesBurned: Int,
    val notes: String?,
    val workoutDate: String, // 格式: "yyyy-MM-dd HH:mm:ss"
    val planId: Long? = null
)